tesseract_core/sdk/cli.py

```diff
@@ -426,6 +426,35 @@ def serve(
             ),
         ),
     ] = None,
+    num_workers: Annotated[
+        int,
+        typer.Option(
+            "--num-workers",
+            help="Number of worker processes to use when serving the Tesseract.",
+            show_default=True,
+        ),
+    ] = 1,
+    propagate_tracebacks: Annotated[
```

Comment:
Could even be called `--unsafe-propagate-tracebacks` or so? Not sure how much of a concern this is in practice.

Reply:
let's leave as is now imho